### PR TITLE
Extract UI from Bot repo for initial skeleton in the new UI repo

### PR DESCRIPTION
**Pull request by nerdalert**
- Cloned the original repository (instructlab-bot) and created a new branch (extract-ui).
- Moved relevant files and directories (ui, .github, templates, lints) into the ui directory.
- Used git filter-branch to retain only the ui directory, making it the root.
- Cleaned up unnecessary history and references.
- Merged new-origin/main into extract-ui with --allow-unrelated-histories to incorporate the initial commit from the new repository.

This process preserves the commit history from the original repository along with Russell's initial commit from the new repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
-# ui
\ No newline at end of file
+# Instructlab UI
```
Comment (markstur) [inline README.md:1]: s/Instructlab/InstructLab/g

Comment (nerdalert) [reply README.md]: fixed, ty.

**Review by Gregory-Pereira — APPROVED** [review]
/lgtm
We should merge this as is, even if there are mistakes. The purpose of this PR is propagating history through to the seed. As such feature changes should come in follow on PRs, we will get more familiar with it as we work it.

**Review by juancappi — APPROVED** [review]

**Review by vishnoianil — APPROVED** [review]
/lgtm

**Pull request merged**